dotCMS/gradle.properties

```diff
@@ -1,6 +1,6 @@
-dotcmsReleaseVersion=master
-coreWebReleaseVersion=next
-webComponentsReleaseVersion=next
```

Comment:
This is what we need in master.

Reply:
done https://github.com/dotCMS/core/pull/23368/commits/2b4582e8d7ff6adaca5c671b58a6f2ef8704b867#diff-01eec14c9d36ad794af257b0869db9c6525d69417c306a7cd2cd9dae64c9710bR1